src/components/Content.jsx

```diff
@@ -0,0 +1,43 @@
+import FeedbackMessage from './FeedbackMessage';
+import FoodBox from './FoodBox';
+import { Divider, Row } from 'antd';
+
+function Content({ food, searchInput, handleClickDelete, getFoodBack }) {
+  return (
+    <>
+      <Divider />
+      <Divider style={{ fontSize: '32px' }} orientation="center">
+        Food List
+      </Divider>
+      {food.length === 0 && <FeedbackMessage {...{ getFoodBack }} />}
+      <Row justify="space-evenly">
+        {food
+          // eslint-disable-next-line array-callback-return
+          .filter((food) => {
+            if (searchInput === '') {
+              return food;
```

Comment:
filter functions should return a boolean value, not the food item itself. i.e. this only works because food is a truthy value, but wouldn't work if it ever were a falsy value.

Reply:
instead this whole statement could be done in a single boolean. 
```js
return searchInput === "" || food.name.toLowerCase().includes(searchInput.trim().toLowerCase())
```  
which could be even shortened to only the second half, because an empty searchInput "" is included in any food.name. i.e. every string includes the empty string.

Reply:
so for filtering you could just do this: 
```js
.filter((food) => food.name.toLowerCase().includes(searchInput.trim().toLowerCase()))
```

Reply:
Awesome!
I didn´t know how to do it and copy/pasted that from some old medium post...